.buildkite/rllib.rayci.yml

```diff
@@ -189,13 +189,13 @@ steps:
     tags: 
       - rllib_gpu
       - gpu
-    parallelism: 2
+    parallelism: 3
     instance_type: gpu-large
     commands:
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}"
         --parallelism-per-worker 2
-        --gpus 4
+        --gpus 6
```

Comment:
this is actually the number of gpus in one machine, so keep it as 4 still ;)

Reply:
Ah, makes sense!! Thanks